crates/metrics/metrics-derive/tests/metrics.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(missing_docs)]
```

Comment:
can we change this to `#![expect(missing_docs)]` so we get a failure when we need to remove these? (ie its fixed upstream)

Reply:
Sure, done.

Reply:
We can. Although, this will trigger a warning.

Reply:
what warning?

Reply:
<img width="682" alt="Screenshot 2024-09-16 at 12 49 59 PM" src="https://github.com/user-attachments/assets/0516185e-cb3c-4848-b994-df5dcb30bd84">

This behaviour is on by default.